litellm/integrations/lunary.py

```diff
@@ -105,19 +105,17 @@ def log_event(
         end_time=datetime.now(timezone.utc),
         error=None,
     ):
-        # Method definition
         try:
             print_verbose(f"Lunary Logging - Logging request for model {model}")
 
             litellm_params = kwargs.get("litellm_params", {})
             optional_params = kwargs.get("optional_params", {})
-            metadata = litellm_params.get("metadata", {}) or {}
+            metadata = litellm_params.get("metadata", {})
 
             if optional_params:
-                # merge into extra
                 extra = {**extra, **optional_params}
 
-            tags = litellm_params.pop("tags", None) or []
+            tags = metadata.get("metadata", None) 
```

Comment:
there's no metadata inside metadata, this will always be None 

is this intended behaviour? @hughcrt

Reply:
sorry, I've pushed the wrong thing. It's fixed in the next commit